lib/iris/coords.py

```diff
@@ -861,7 +864,8 @@ def xml_element(self, doc):
                 element.setAttribute(
                     "climatological", str(self.climatological)
                 )
-
+        if self.ignore_axis:
+            element.setAttribute("ignore_axis", str(self.ignore_axis))
```

Comment:
```suggestion
        if hasattr(self, "ignore_axis") and self.ignore_axis:
            element.setAttribute("ignore_axis", str(self.ignore_axis))
```

Reply:
Actually ... thinking on this, I don't believe that we should capture the state of `ignore_axis` within the CML. We're not interested in stateful equality here, and we've got separate unit test coverage to give us confidence that the new behaviour is working as expected.

So yeah, I vote from removing this. I genuinely don't see the benefit from the effort of extending the CML here.

@HGWright Thoughts?